docs/src/Theory/Atmos/Microphysics.md

```diff
@@ -108,7 +108,7 @@ The default value of ``C_{drag}`` is chosen such that the ``v_t`` is
 close to the empirical terminal velocity formulation in Smolarkiewicz
 and Grabowski 1996. Assuming a constant drag coefficient is
 an approximation as it should be size and flow dependent, see
-[drag_coefficient](https://www.grc.nasa.gov/www/K-12/airplane/dragsphere.html).
+[drag coefficient](https://en.wikipedia.org/wiki/Drag_coefficient).
```

Comment:
For some reason, the nasa link was causing a link-check warning. We shouldn’t change the url for this reason, but I did this for a quick fix.